style.css

```diff
@@ -1,1022 +1,802 @@
 }
 
-.footermaintext {
-  width: 20.4375rem;
-  left: 1.5rem;
-  font-family: Poppins, Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 1.25rem;
-  text-align: center;
-  margin-top: 1rem;
+.forbackgroudn {
+  background-image: url(./image/Newcontact\ form\ background\ shapes@2x.png);
+  z-index: 10px;
 }
 
-.footertext {
-  font-family: Poppins, Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 1.25rem;
-  padding-left: 0.1rem;
-  text-align: center;
-  color: #ebebff;
+.contact {
+  padding-top: 100px;
+  background: #7b86eb;
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-top-left-radius: 5rem;
 }
 
-.contact {
+.contact .sub-contact {
   display: flex;
-  align-items: center;
+  flex-direction: column;
   text-align: center;
-  margin-top: 3rem;
-  height: 3.5rem;
-}
-
-.hifooter {
-  width: 18rem;
-  height: 3.5rem;
-  font-family: Poppins, Arial, Helvetica, sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 2.5rem;
-  line-height: 3.25rem;
-  color: #fff;
+  align-items: center;
+  padding: auto 180px 50px 180px;
 }
 
-form {
+.contact form {
   display: flex;
   flex-direction: column;
-  width: 90%;
-  padding: 1.125rem;
-  margin-bottom: 1.25rem;
+  padding-top: 20px;
+  padding-left: 18px;
+  padding-bottom: 18px;
+  padding-right: 18px;
+  margin-bottom: 20px;
+  align-items: center;
+  gap: 1rem;
 }
 
-form input[type="text"],
-input[type="email"],
-textarea[type="area"] {
-  border-radius: 5px;
+.contact form textarea[type="area"] {
+  width: 342px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
   padding: 15px;
-  border: none;
-  margin: 7px;
+}
+
+.contact form input[type="text"],
+input[type="email"] {
+  padding: 15px;
+  width: 342px;
+  height: 41px;
   color: #8300ff;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 2px;
 }
 
 input[type="submit"] {
   border-radius: 5px;
-  padding: 15px;
+  padding: 10px;
   border: none;
   margin: 7px;
+  width: 100px;
   color: #6070ff;
-  width: 50%;
-  font-size: 21px;
+  font-size: 12px;
   font-weight: 400;
-  transition: 0.5s ease;
-  background: #fff;
+  transition: all 0.5s ease-out;
 }
 
-input[type="submit"]:hover {
+input[type="submit"]:hover,
+input[type="submit"]:focus {
   background-color: #396df2;
   color: #fff;
 }
 
-.footermaintext p {
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  text-align: center;
-  color: #ebebff;
-  margin-bottom: 25px;
-  margin-left: 1.4rem;
-}
-
-.button_form:hover {
-  background: #6070ff;
-  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  border-radius: 8px;
-  color: white;
-}
-
-.button_form:active {
-  background: #2230d2;
-  color: white;
-}
-
-.button_form:enabled {
-  border: 1px solid #7f8cff;
-  border-radius: 8px;
-}
-
-.button_form:disabled {
-  background: #e5e5e5;
+@media only screen and (min-width: 768px) {
```

Comment:
- [ OPTIONAL ] I will suggest that you fix your mobile design styles. You just need to keep all the styles for mobile which are above this media query intact and add your desktop styles in this media query that's all and your project should look good on both desktop and mobile  :100: 
- Currently, you have deleted most of the styles for the mobile view but kindly note that this is why you were introduced to media queries so that all you need to do is to add new styles to adapt your design to the desktop without messing up the original styles for your mobile view. :+1: